google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurableTest.java

```diff
@@ -43,7 +43,7 @@
   private TextFieldWithBrowseButton cloudSdkDirectoryField;
 
   private static final String CLOUD_SDK_DIR_PATH = new File("/a/b/c/gcloud-sdk").getAbsolutePath();
-  private static final String MISSING_SDK_DIR_WARNING = "Please select a Cloud SDK home directory.";
+  private static final String MISSING_SDK_DIR_WARNING = "Cloud SDK home directory is not correct.";
```

Comment:
My fault. Its the unit test variables that are named poorly. Here's what is actually in our bundle:

```
appengine.cloudsdk.location.invalid.message=Cloud SDK home directory is not correct.
appengine.cloudsdk.location.missing.message=Cloud SDK home directory is not specified.

```
Let me know if those look ok.